waargbench/waargbench.cabal

```diff
@@ -65,21 +65,9 @@ benchmark bench
                      , attoparsec   >= 0.13   && < 0.15
                      , hw-json      >= 0.9    && < 0.10
                      , scientific   >= 0.3    && < 0.4
-                     , gauge
+                     , criterion
                      , waargonaut
 
   hs-source-dirs:      src
   default-language:    Haskell2010
-  ghc-options:         -Wall -O2
-
--- benchmark bench
```

Comment:
This can be moved to a separate cabal file so it is runnable but won't be built by default